scripts/LPSRE/increasing-observability-operator-pvc-size/script.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+# This job outlines the procedure for increasing the PVC size of an RHACS (Red Hat Advanced Cluster Security) instance. 
+# It includes steps for inspecting logs, retrieving the current PVC, editing the PVC size, and verifying the resize process.
+set -euxo pipefail
+
+if [[ -z "${NEW_SIZE}" ]]; then
```

Comment:
small suggestion, maybe validate the NEW_SIZE before apply to avoid human error.
like, the NEW_SIZE should be larger than existing, or the given format is correct via regex.

Reply:
makes sense, I've added both format check using regex and size check. Though calculating the sizing is bringing some overhead to the readability